options/groupeditor.js

```diff
@@ -262,8 +262,11 @@ var groupEditor = (() => {
     },
     onSearchInput(event) {
       let api = urlsGridOptions.api;
-      api.setQuickFilter(document.getElementById('urlsSearchBar').value);
-      editor.disableUrlsButtons();
+      let searchBarElem = document.getElementById('urlsSearchBar');
+      console.log('test1');
```

Comment:
Please remove this log.

Reply:
My mistake, I missed this somehow when looking at the git diff. Should be fixed now.